src/main/resources/application.yml

```diff
@@ -0,0 +1,7 @@
+spring:
+  profiles:
+    active: default
```

Comment:
See: https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-profiles

Default is the profile that is active by default by spring. Setting this explicitly here for clarity